transport_plugins/bled112/iotile_transport_bled112/virtual_bled112.py

```diff
@@ -501,3 +593,17 @@ def _send_trace(self, chunk=None):
                 traceback.print_exc()
                 print("*** END EXCEPTION ***")
                 self._audit('ErrorStreamingTrace')  # If there was an error, stop streaming but don't choke
+
+
+    #FIXME Need method to set these values
```

Comment:
I don't think these methods are needed since the broadcast value is already updated when the virtual device streams a broadcast reading.